stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -35,6 +37,7 @@ module SwiftGlibc [system] {
       header "${GLIBC_INCLUDE_PATH}/complex.h"
       export *
     }
+% if CMAKE_SDK != "FREEBSD":
     module pty {
       header "${GLIBC_INCLUDE_PATH}/pty.h"
```

Comment:
How about including `libutil.h` instead?
(And for `utmp.h` instead including `utmpx.h`?)